demo/content/Text.js

```diff
@@ -18,25 +18,25 @@ export default class DemoContentText extends Component {
    constructor(props) {
       super(props)
 
-      this.handleEditCancel = this.handleEditCancel.bind(this)
-      this.handleEditDone = this.handleEditDone.bind(this)
+      this.handleCancel = this.handleCancel.bind(this)
```

Comment:
Tbh I like the original language, seems clearer to me (eg. `handleEditCancel`). Thoughts?